bot/seasons/evergreen/minesweeper.py

```diff
@@ -0,0 +1,198 @@
+import typing
+from random import random
+
+import discord
+from discord.ext import commands
+
+
+class Minesweeper(commands.Cog):
+    """Play a game of minesweeper."""
+
+    def __init__(self, bot: commands.Bot) -> None:
+        self.bot = bot
+        self.games: typing.Dict[discord.member, typing.Dict] = {}  # Store the currently running games
```

Comment:
Could this typing be stored as a variable at the top of the file and named something like DictOfGames to avoid the long typing and comment explaining it here?

Reply:
Bonus question, why not ID over member for key?  Just curious

Reply:
just tough it would be easier, would id be better?

Reply:
I'm unsure how a member is hashed, so it might change, ids dont

Reply:
okay then